src/components/grid/index.stories.mdx

```diff
@@ -0,0 +1,102 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Box } from '@chakra-ui/react';
+
+import Grid from './index.tsx';
+
+<Meta
+  title="Layout/Grid"
+  component={Grid}
+  parameters={{
+    controls: {
+      sort: 'alpha',
+    },
+  }}
+  args={{
+    columns: 3,
+    spacing: 'md',
+  }}
+  argTypes={{
```

Comment:
Do we really need that many possible combinations for your grids?

Reply:
It's spacing, columns and the auto-responsive variant. Is that really too many props for the flexibility we get? If yes - what would you cut?

Reply:
I meant more the other way around. If we wanna expose that big of flexibility as I assume we not gonna need that many combinations of the grid in the applications where we can have a grid of any size and spacing...

Reply:
It uses values from the theme (okay not only, but that's the case with all chakra components and I think it's fine). 
I tend to be on the side that the layout component should have more flexibility - we would use it to build bigger and more complex layouts after all.